javaparser-core-testing/src/test/java/com/github/javaparser/ast/comments/CommentTest.java

```diff
@@ -21,23 +21,29 @@
 
 package com.github.javaparser.ast.comments;
 
+import static com.github.javaparser.StaticJavaParser.parse;
+import static com.github.javaparser.utils.TestUtils.assertEqualsStringIgnoringEol;
+import static com.github.javaparser.utils.Utils.SYSTEM_EOL;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
 import com.github.javaparser.ast.CompilationUnit;
 import com.github.javaparser.ast.NodeList;
 import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
 import com.github.javaparser.ast.body.MethodDeclaration;
 import com.github.javaparser.javadoc.Javadoc;
 import com.github.javaparser.javadoc.description.JavadocDescription;
-import com.github.javaparser.printer.PrettyPrinterConfiguration;
-import org.junit.jupiter.api.Test;
-
-import static com.github.javaparser.StaticJavaParser.parse;
-import static com.github.javaparser.utils.TestUtils.assertEqualsStringIgnoringEol;
-import static com.github.javaparser.utils.Utils.SYSTEM_EOL;
-import static org.junit.jupiter.api.Assertions.*;
+import com.github.javaparser.printer.configuration.ConfigurablePrinter;
+import com.github.javaparser.printer.configuration.Indentation;
+import com.github.javaparser.printer.configuration.Indentation.IndentType;
+import com.github.javaparser.printer.configuration.PrinterConfiguration;
 
 class CommentTest {
 
-    private static final PrettyPrinterConfiguration PRETTY_PRINTER_CONFIG_TWO_INDENT = new PrettyPrinterConfiguration().setIndentSize(2);
+    private static final ConfigurablePrinter PRETTY_PRINTER_CONFIG_TWO_INDENT = new PrinterConfiguration().setIndentation(new Indentation(IndentType.SPACES, 2));
```

Comment:
A printer configuration is a configurable printer? Eh?

Reply:
I am open to all naming proposals

Reply:
I will rename ConfigurablePrinter to PrinterConfiguration (configuration of a Printer) and the actual implementation PrinterConfiguration to DefaultPrettyPrinterConfiguration (because PrettyPrinterConfiguration already exists)

Reply:
Configurableprinter interface is renamed PrinterConfiguration